sys-freebsd/freebsd-lib/files/freebsd-lib-add-nossp-cflags.patch

```diff
@@ -0,0 +1,30 @@
+Please do not disable this patch.
```

Comment:
Oh, here's an explanation but not a good one. Focus on what the patch changes, no need for big fat warnings when patch is described correctly.